docs/src/content/features/styling.mdx

```diff
@@ -0,0 +1,465 @@
+# Styling
+
+<Subtitle>Learn how to style your application using Pigment CSS.</Subtitle>
+<Meta name="description" content="Learn how to style your application using Pigment CSS." />
+
+> [!IMPORTANT]
+> Pigment CSS is currently in alpha stage and the APIs are subject to change. Each change will be recorded in the [releases](/overview/releases) page.
+> Please report any issues you face on our [GitHub](https://github.com/mui/pigment-css).
+
+After you have [configured your bundler](/overview/integrations), you can start styling your application using Pigment CSS.
+
+## css
+
+This is the most basic API in Pigment CSS that allows you to style your application using CSS in a framework agnostic way.
+
+You can use either the tagged-template syntax of the JS object syntax to style your application. You can also mix and match the two as per the needs.
+
+### Tagged-template syntax
```

Comment:
I'm likely going to use only one of the syntaxes exclusively in my application. Seeing the other one in the dosc feels a bit like clutter. Would it make sense to turn this into a selector like the "tailwind css"/"css modules" one from the base ui docs?